install.sh

```diff
@@ -298,6 +298,7 @@ nvm_do_install() {
   INSTALL_DIR="$(nvm_install_dir)"
 
   SOURCE_STR="\nexport NVM_DIR=\"$INSTALL_DIR\"\n[ -s \"\$NVM_DIR/nvm.sh\" ] && \\. \"\$NVM_DIR/nvm.sh\"  # This loads nvm\n"
+  SOURCE_STR2="[ -s \"\$NVM_DIR/bash_completion\" ] && [ -n \"\$BASH_VERSION\" ] && [ -n \"\$ZSH_VERSION\" ] && \\. \"\$NVM_DIR/bash_completion\"  # This loads nvm bash_completion\n"
```

Comment:
Okay, I think I understand your point now, I do that because of the last few lines:
```sh
  echo "=> Close and reopen your terminal to start using nvm or run the following to use it now:"                                                                            
  command printf "$SOURCE_STR"                                                                                                                                               
  command printf "$SOURCE_STR2"
```

I may need to you suggestion on this part, unless you don't mind me to write a case switch here again.